monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -83,3 +92,12 @@ def delete(self, file_type):
             ConfigService.set_config_value(filename_path, "")
 
         return {}
+
+    @staticmethod
+    def check_file_type(file_type):
+        """
+        Check if the file type is not supported
```

Comment:
Rather then explaining what method does through a comment make the name clearer. Instead of `check_file_type` do `is_pba_file_type_supported`.